src/components/Error/Error.js

```diff
@@ -0,0 +1,20 @@
+// Generic error component
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Wrapper from './Wrapper';
+
+function Error({ message }) {
+  return <Wrapper role="alert">{message}</Wrapper>;
+}
+
+Error.defaultProps = {
```

Comment:
I propose to go with ES6 Default Parameters for Functional components instead of `defaultProps`

```
const DEFAULT_MESSAGE = 'Oops, something went wrong.';

function Error({ message = DEFAULT_MESSAGE }) {
  return <Wrapper role="alert">{message}</Wrapper>;
}

```

Reply:
👍